README.md

```diff
@@ -2,9 +2,223 @@
 
 [![Build Status](https://travis-ci.org/revdotcom/revai-java-sdk.svg?branch=develop)](https://travis-ci.org/revdotcom/revai-java-sdk)
 
+## Documentation
+
+See the [API docs](https://www.ai.rev.ai/docs) for more information about the API.
+
+## Install the SDK
+The recommended way to use the Rev.ai Java SDK is to import it into the project using Maven.
+
+      <dependency>
+        <groupId>ai.rev</groupId>
+        <artifactId>revai-java-sdk</artifactId>
+        <version>1.0.0</version>
+      </dependency>
+    
+## Installing it locally
+Once you've cloned the repo you can use Maven to build it locally and install it your local Maven .m2 repository.
+
+    mvn install -Dmaven.test.skip=true
+
+## Support
+
+We support Java 8 and 11.
+
+## Usage
+
+All you need to get started is your Access Token, which can be generated on
+your [Settings Page](https://www.ai.rev.ai/access_token). Create a client with the
+given Access Token:
+
+```
+// Initialize your client with your ai.rev access token
+String accessToken = "Your Access Token";
+ApiClient apiClient = new ApiClient(accessToken);
+```
+
+### Checking credits remaining
+
+```
+RevAiAccount revAiAccount = apiClient.getAccount(); 
+```
+
+### Submitting a job
+
+You can submit a local file
+```
+RevAiJob revAiJob = apiClient.submitJobLocalFile("./path/to/file.mp3");
+```
+
+or submit via a public url
+```
+RevAiJob revAiJob = apiClient.submitJobUrl("https://www.ai.ai.rev.ai/FTC_Sample_1.mp3");
```

Comment:
is that a real media url?

Reply:
I used to be lol. It got messed up in a refactor. I thought I found all of those. Good catch!